src/solver/conv_asm_implicit_gemm_wrw_gtc_dynamic_xdlops.cpp

```diff
@@ -167,6 +167,220 @@ GetImplicitGemmWrwGTCDynamicXdlopsKernelList()
+        { "wrw", "fp16",   4,   0,  32, 256,  32,   4,  64,   4,   2,   1,   2,   2,   {1,   4,   1,   1},   {1,   8,   1,  32},   {1,   4,   8,   1},   {1,   8,   1,  32},   1},
+        { "wrw", "fp16",   4,   0,  32, 256,  32,   4,  64,   4,   2,   1,   2,   2,   {1,   4,   1,   1},   {1,   8,   1,  32},   {1,   4,   8,   1},   {1,   8,   1,  32},   0},
+        { "wrw", "fp16",   4,   0,  32, 256,  16,   4,  64,   4,   2,   1,   2,   2,   {1,   2,   1,   1},   {1,   8,   1,  32},   {1,   2,   8,   1},   {1,   8,   1,  32},   1},
+        { "wrw", "fp16",   4,   0,  32, 256,  16,   4,  64,   4,   2,   1,   2,   2,   {1,   2,   1,   1},   {1,   8,   1,  32},   {1,   2,   8,   1},   {1,   8,   1,  32},   0},
+        { "wrw", "fp16",   4,   1,  32, 256,  32,   4,  64,   4,   2,   1,   2,   2,   {1,   1,   1,   4},   {1,  32,   1,   8},   {1,   1,  32,   1},   {1,  32,   1,   8},   1},
+        { "wrw", "fp16",   4,   1,  32, 256,  32,   4,  64,   4,   2,   1,   2,   2,   {1,   1,   1,   4},   {1,  32,   1,   8},   {1,   1,  32,   1},   {1,  32,   1,   8},   0},
+        { "wrw", "fp16",   4,   1,  32, 256,  16,   4,  64,   4,   2,   1,   2,   2,   {1,   1,   1,   2},   {1,  16,   1,  16},   {1,   1,  16,   1},   {1,  16,   1,  16},   1},
+        { "wrw", "fp16",   4,   1,  32, 256,  16,   4,  64,   4,   2,   1,   2,   2,   {1,   1,   1,   2},   {1,  16,   1,  16},   {1,   1,  16,   1},   {1,  16,   1,  16},   0},
+        { "wrw", "fp16",  16,   1,  32, 256,  16,   4,  64,   4,   2,   1,   2,   2,   {1,   1,   1,   2},   {1,  16,   1,  16},   {1,   1,  16,   1},   {1,  16,   1,  16},   1},
+        { "wrw", "fp16",  16,   1,  32, 256,  16,   4,  64,   4,   2,   1,   2,   2,   {1,   1,   1,   2},   {1,  16,   1,  16},   {1,   1,  16,   1},   {1,  16,   1,  16},   0},
+        { "wrw", "fp16",   4,   0,  64,  64,  32,  16,  16,  16,   1,   1,   2,   2,   {1,   4,   1,   2},   {1,   8,   1,  32},   {1,   4,   2,   1},   {1,   8,   1,  32},   1},
+        { "wrw", "fp16",   4,   0,  64,  64,  32,  16,  16,  16,   1,   1,   2,   2,   {1,   4,   1,   2},   {1,   8,   1,  32},   {1,   4,   2,   1},   {1,   8,   1,  32},   0},
+        { "wrw", "fp16",   4,   0,  64,  64,  16,  32,  32,   8,   1,   1,   1,   1,   {1,   4,   1,   1},   {1,   4,   1,  64},   {1,   4,   1,   1},   {1,   4,   1,  64},   1},
+        { "wrw", "fp16",   4,   0,  64,  64,  16,  32,  32,   8,   1,   1,   1,   1,   {1,   4,   1,   1},   {1,   4,   1,  64},   {1,   4,   1,   1},   {1,   4,   1,  64},   0},
+        //{ "wrw", "fp16",   4,   1,  64,  64,  32,  16,  16,  16,   1,   1,   2,   2,   {1,   1,   1,   8},   {1,  32,   1,   8},   {1,   1,   8,   1},   {1,  32,   1,   8},   1},
+        //{ "wrw", "fp16",   4,   1,  64,  64,  32,  16,  16,  16,   1,   1,   2,   2,   {1,   1,   1,   8},   {1,  32,   1,   8},   {1,   1,   8,   1},   {1,  32,   1,   8},   0},
```

Comment:
Any specific reason for commenting this out?

Reply:
Because these 2 kernels with tile size 64x64x32 has lower efficiency than 64x64x16 kernels.

Reply:
[Recommendation] Can you please add relevant comment?

Reply:
OK, added.